src/main/java/esi/backend/model/Customer.java

```diff
@@ -31,6 +31,54 @@ public class Customer extends User {
     private Set<Invoice> invoices = new HashSet<>();
 
 
+    public String getFirstName() {
```

Comment:
Do we need to add an endpoint so it would be possible to change these values through the API? For example a method for PUT /customers/{customerId}

Reply:
We might try, but I'm not sure if we manage before midnight.